docs/contributor/04-10-workflows.md

```diff
@@ -95,3 +95,59 @@ The workflow performs the following actions:
 - Waits for the binary image to be ready in the registry
 - Installs the module
 - Runs the E2E BTP Manager secret customization test on the cluster
+
+### Performance Tests
+
+The [workflow](/.github/workflows/run-performance-tests-reusable.yaml) runs performance and tests on the k3s cluster. The following parameters are required from the calling workflow:
+
+| Parameter name       | Required | Description                     |
+|----------------------|----------|---------------------------------|
+| **image-repo**       | yes      | Binary image registry reference |
+| **image-tag**        | yes      | Binary image tag                |
+| **credentials-mode** | yes      | Specifies whether to use real or dummy credentials |
+
+The workflow performs the following actions for all jobs:
+- Prepares the k3s cluster with the Docker registry
+- Waits for the binary image to be ready in the registry
+- Installs the module
+
+#### Jobs
+1. **Frequent Secret Update Test**
+- **Purpose**: Evaluates the system's response time and reconciliation success rate when the `sap-btp-manager` secret is updated frequently.
+- **Steps**:
+    - Patches the `sap-btp-manager` secret in a loop to simulate frequent updates.
+    - Fetches metrics from the `btp-manager-controller-manager` to measure average reconcile time, reconcile errors, and other reconciliation statistics.
+- **Failure Handling**:
+    - The test fails if the average reconcile time exceeds the defined threshold.
+    - The test fails if any reconcile errors are detected.
+
+2. **Reconcile Secret Deletion Test**
+- **Purpose**: Measures the reconciliation performance of BtpOperator when the `sap-btp-manager` secret is repeatedly deleted and reapplied.
```

Comment:
```suggestion
- **Purpose**: Measures the reconciliation performance of BtpOperator when the `sap-btp-manager` Secret is repeatedly deleted and reapplied.
```

Reply:
Why not use `the SAP BTP service operator` instead of `BtpOperator`? Especially as in some sentences you also mention `BTP Manager` (and not `btp-manager`)

Reply:
SAP BTP service operato != BtpOperator. We are checking how the BtpOperator CR is reconciled.

Reply:
But now I think that we should use here the name of the module (BTP Manager)

Reply:
My question only refers to the choice of name: technical vs. ordinary language.
The choice is between BTP Manager and the SAP BTP service operator vs. btp-manager and BtpOperator.

Reply:
SAP BTP service operator is not BtpOperator. We have a custom resource that is part of BTP Manager and its called BtpOperator.